src/doc/en/developer/github.rst

```diff
@@ -73,6 +73,18 @@ and check::
 
 which will show the default repo along with its readme, which is quite long.
 
+``gh`` extensions
+-----------------
+
+``gh`` is extendable; e.g.  a useful extension to ``gh`` allows testing of
+Sage's GitHub Actions locally, using Docker. It is called ``act`` and can be
+installed by running
+
+    [alice@localhost sage]$ gh extension install https://github.com/nektos/gh-act
+
+Append ``--force`` flag to force an upgrade.
```

Comment:
Please check the built documentation.

`... by running ::`?

Reply:
maybe add a note saying "this section is a stub" ?

Reply:
Sorry, I don't understand. 

If you want to keep it, then at least fix `... by running ::`.

Reply:
ok

Reply:
fixed by [6a6e5ec](https://github.com/sagemath/sage/pull/36117/commits/6a6e5ec0dab46344e6b1c116da1add5b0177b2b1)